optimade/server/entry_collections.py

```diff
@@ -16,10 +16,10 @@
 
 if CONFIG.use_real_mongo:
     from pymongo import MongoClient
+    client = MongoClient(CONFIG.mongo_uri)
 else:
     from mongomock import MongoClient
-
-client = MongoClient()
+    client = MongoClient()
```

Comment:
Is this not a possibility in `mongomock`? Or maybe it just doesn't make sense?

Reply:
It actually works fine as far as code goes, but it's probably not guaranteed behavior, so it doesn't hurt to separate the two unless we're getting issues elsewhere like linting.

Reply:
It was merely for the elegance of the code as it was - only having one line for the instantiation of the client - but never mind.